Simplified/Reader2/NYPLBookLocation+Locator.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  NYPLBookLocation+Locator.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2020-11-09.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import R2Shared
+
+extension NYPLBookLocation {
+  static let r2Renderer = "readium2"
+  
+  convenience init?(locator: Locator, publication: Publication, renderer: String) {
+    guard let idref = publication.idref(forHref: locator.href) else {
+      Log.warn(#file, "Unable to find Publication's idref from locator's idref - \(locator.href)")
```

Comment:
@ettore Since you are removing `Log.warn`, should I change this to `debug`? And the one on line 32 too

Reply:
actually, sorry, I am not removing it. I spoke too soon!